ci-operator/step-registry/aro-hcp/lease/release/aro-hcp-lease-release-commands.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# shellcheck disable=SC1091
+source "$LEASE_PROXY_CLIENT_SH"
+source ci-operator/step-registry/aro-hcp/lease/common/aro-hcp-lease-common-commands.sh
```

Shellcheck (0.11.0)

[info] Not following: ci-operator/step-registry/aro-hcp/lease/common/aro-hcp-lease-common-commands.sh was not specified as input (see shellcheck -x).

(SC1091)